lib/casino/ldap_authenticator.rb

```diff
@@ -77,17 +77,16 @@ def user_filter(username)
   end
 
   def extra_attributes(user_plain)
-    if @options[:extra_attributes]
```

Comment:
Actually, there's an issue about this exact thing https://github.com/rbCAS/casino-ldap_authenticator/issues/9